packages/lisk-api-client/test/api_resource.ts

```diff
@@ -134,7 +133,8 @@ describe('API resource module', () => {
 						data: sendRequestResult,
 					},
 				});
-				return resource.request(defaultRequest, true).catch(err => {
+
+				return resource.request(defaultRequest, false).catch(err => {
```

Comment:
why change the repeat flag? (also below)

Reply:
with the repeat=true it was not actually reaching the catch block and thus err.message was not really being tested.